frontend/src/components/poster/Poster.jsx

```diff
@@ -6,6 +6,7 @@ function Poster() {
       className="poster__img"
       src="https://i.ebayimg.com/images/g/HDwAAOSwFJZkvjiG/s-l1600.jpg"
       alt=""
+      draggable="false"
```

Comment:
Why draggable attribute ?

Reply:
Without this attribute, we can drag the image and the blur disappears. It is an anti-cheat.